src/Components/Buttons/IncrementDecrementBtn.js

```diff
@@ -0,0 +1,29 @@
+import { useCart } from "../Context/CartContext";
+
+export default function IncDecBtn({ item }) {
+  const { dispatch } = useCart();
+  return (
+    <div className="incrementDecrementContainer">
+      <button
```

Comment:
You can change the order of "increment" and "decrement" buttons, since the `+` button on the left side doesn't look natural.

Reply:
Changed the order of buttons